packages/inter-protocol/src/vaultFactory/vault.js

```diff
@@ -179,6 +178,42 @@ export const VaultI = M.interface('Vault', {
 export const prepareVault = (baggage, marshaller, zcf) => {
   const makeVaultKit = prepareVaultKit(baggage, marshaller);
 
+  const AdjustBalancesProposalShape = M.splitRecord({
```

Comment:
```suggestion
  // Adjustments must be able to specify just the 'give' or the 'want'.
  // There's no use case for giving both brands at once, but it allows
  // for the above requirement and is supported by the contract
  const AdjustBalancesProposalShape = M.splitRecord({
```

Reply:
There is a use case for giving both at once. If you want to rescue a vault that's on the verge of being liquidated, and you have limited resources, you might add collateral at the same time that you're repaying IST.

Reply:
In light of @Chris-Hibbert 's comment and @turadg thumbs-up agreement, kept just the first line of the comment. Also added that line to the `AdjustBalancesProposalShape` which is now similar to this one.

Done.